### PR TITLE
treetime: add page

### PR DESCRIPTION
**Pull request by vatana7**
- [ x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ x] The page(s) have at most 8 examples.
- [ x] The page description(s) have links to documentation or a homepage.
- [ x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/treetime.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# treetime
+
+> TreeTime provides routines for ancestral sequence reconstruction and inference of molecular-clock phylogenies.
+> More information: <https://manned.org/treetime>.
+
+- Infer ancestral sequences maximizing the joint or marginal likelihood:
+
+`treetime ancestral`
```
Comment (marchersimon) [inline pages/linux/treetime.md:8]: I have no idea what any of this means, but what exactly happens when you run that command? Does it have to be executed inside a project? Does this print something? Does this open some interactive prompt?

Comment (vatana7) [reply pages/linux/treetime.md:8]: Yes it's supposed to print out this. Example: 
image

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.
> Also, Welcome to tldr btw.

File: pages/linux/treetime.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# treetime
+
+> TreeTime provides routines for ancestral sequence reconstruction and inference of molecular-clock phylogenies.
+> More information: <https://manned.org/treetime>.
```
Comment (kbdharun) [inline pages/linux/treetime.md:4]:
```suggestion
> More information: <https://treetime.readthedocs.io/en/latest/tutorials.html>.
```
Just a small suggestion, I found their official documentation can you update your link to this?

**Review by kbdharun — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @vatana7

**Pull request merged**